packages/components/empty/src/instance.ts

```diff
@@ -0,0 +1,3 @@
+import type Empty from './empty.vue'
+
+export type EmptyInstance = InstanceType<typeof Empty>
```

Comment:
Why move it to a new file? Is there any special consideration?

Reply:
Thank you for the review!

## reference

https://github.com/element-plus/element-plus/pull/10151/files#diff-9966b90b33c39a3a31e544fcd2f7f1da6c78f4b051d829ae847fde21ce019de0

Note that instance is separated into a separate file!